src/drop-down-menu.jsx

```diff
@@ -343,7 +359,12 @@ const DropDownMenu = React.createClass({
   },
 
   _onMenuRequestClose() {
-    this.setState({open:false});
+    setTimeout(() => {
```

Comment:
Sounds like we have an important issue then. I would rather find it.
Adding the setTimeout is triggering to render when clicking on an item (following this line https://github.com/chrismcv/material-ui/blob/dropdown-popover-menu/src/drop-down-menu.jsx#L351)